.golangci.yml

```diff
@@ -24,16 +24,16 @@ linters:
     - govet
     - ineffassign
     # - interfacer
-    - lll
+    # - lll
```

Comment:
While we're here, we should just delete the commented out ones.

Reply:
Done.